functions/z.fish

```diff
@@ -0,0 +1,15 @@
+function z
+    if test (count $argv) -gt 0
+        set _Z_RESULT (zoxide query $argv)
+        switch "$_Z_RESULT"
+            case 'query: *'
+                cd (string sub -s 8 -- "$_Z_RESULT")
+                commandline -f repaint
+            case '*'
+                if test -n "$_Z_RESULT"
```

Comment:
Instead of checking for a non-empty string, why not use `echo -n` instead? That way, if the string is empty, it won't output anything.

Reply:
Yep, that works, thanks for the idea!